deployments/charts/penumbra-node/templates/statefulset.yaml

```diff
@@ -30,6 +30,12 @@ spec:
     {{- if eq .Values.cometbft.config.indexer "psql" }}
     - metadata:
         name: db
+        labels:
+          {{- include "penumbra-node.labels" . | nindent 10 }}
+          "app.kubernetes.io/component": fullnode
+          {{- if .Values.part_of }}
+          "app.kubernetes.io/part-of": {{ .Values.part_of }}
+          {{- end }}
```

Comment:
The labels here always should have been present, but weren't. This change won't apply cleanly to running indexers, but that's fine: I need to recreate them anyway after this logic lands. I'll handle that manually.